### ISSUE TITLE
 buzzing noise and  too bright 

### ISSUE THREAD

**Issue by Cyrus331** [skumlos/pcengine-rgb-addon#1]
I made a work based on your open source file. When I tested it, I found that it was too bright whether using AV or RGB output. Moreover, when using RGB output, the sound had a buzzing noise. When using AV, there was no background noise, and the AV output The screen is not only too bright but also white. Later, after I added 75R to the video terminal lead-out on the drawing, the over-brightness improved, but it was still white. Later, after looking at the open source drawings of other brothers, I found that there seemed to be a problem with the video and SYNC marks on the PCB. It should be reversed. I wonder if this is a problem related to making a living.

**Comment by skumlos** [comment]
Hi,
buzzing noise is not a problem I've been noticing, but this will most likely be very dependent on your cable. In any case, if you checkout the "rev-c" branch instead of master, it contains a bunch of changes, mostly related to sync and signal attenuation, but also to support CD expansion audio.

**Comment by donly** [comment]
Hello, @skumlos.
I tried both versions and found that the `master`'s picture is brighter than the `rev-c`, and the sound is also louder, maby something wrong on my boards.

**Comment by skumlos** [comment]
Yeah the master branch is definitely brighter as the input to the THS7374 isnt properly attenuated. Audio *should* be the same, though I am unsure if the expansion audio stuff changes anything here (it shouldnt tho to my knowledge).

**Comment by Cyrus331** [comment]
I swapped video_RAM and /SYNC and added a 75R resistor to the RGB signal output. After that, the picture became normal. It was no longer bright and the colors were normal. There was just some noise and vertical bars. The vertical bars were changed to 133 and 135. The capacitance is slightly weakened, but still there. In addition, I tried grafting an RGB to SVEDIO module to the output of the 7374. It seems that it has not been successful so far. The resulting picture is black and white, distorted and jumpy. Maybe the RGB signal should be taken directly from the tail plug terminal..
![微信截图_20231106103317](https://github.com/skumlos/pcengine-rgb-addon/assets/98383215/15c6e637-1941-4ca4-905a-534d36ead000)

**Comment by donly** [comment]
> Yeah the master branch is definitely brighter as the input to the THS7374 isnt properly attenuated. Audio _should_ be the same, though I am unsure if the expansion audio stuff changes anything here (it shouldnt tho to my knowledge).

When I disable the `CD expansion audio` with disconnect `C3` pin, the sound works on my White PC-Engine(`PI-TG001`) finally.
Does the White PC-Engine support CD audio expansion, I find this thread [White PC-Engine RGB Mod CD-Audi](https://nfggames.com/forum2/index.php?topic=1755.0) but still don't know how to use it.

**Comment by Cyrus331** [comment]
Hello, after I re-corrected the PCB, all the problems were solved. There was no noise or vertical bars. It was very good. I lent it to a friend of mine for testing. He said that when he used the Krikzz Turbo Everdrive Pro to play CD games There is no sound when playing, does this support CD audio? Or did I not make it well?
